templates/MMM-NHL.njk

```diff
@@ -15,13 +15,29 @@
             {% for index in range(rotateIndex, maxGames) %}
                 <tr>
                     <td>{{ games[index] | calendar }}</td>
-                    <td class="align-right">{{ games[index].teams.home.short }}</td>
-                    <td><img class="icon {{ "no-color" if not config.colored }}" src="https://assets.nhle.com/logos/nhl/svg/{{ games[index].teams.home.short }}_light.svg"/></td>
+                    <td class="align-right">
+                    {% if config.showNames %}
```

Comment:
I think it is cleaner to move the if condition outside of the td element. It also makes the module width a bit smaller, which was your main concern anyway, because the empty column doesn't add table spacing this way. I made a screenshot as a comparison.

If inside td element
![image](https://user-images.githubusercontent.com/9334168/104952027-fe9b0880-59c3-11eb-8c03-4569226d52da.png)

If outside of td element
![image](https://user-images.githubusercontent.com/9334168/104952143-3904a580-59c4-11eb-8cde-afa4db771032.png)

Additionally in line 9 and 11 we should compute the colspan attribute based on the config options sth like `colspan="{{ 1 + (1 if config.showNames else 0) + (1 if config.showLogos else 0) }}"`

Reply:
100% agree, just me being lazy because of the colspans